tests/stdlib/tunroll.nim

```diff
@@ -0,0 +1,9 @@
+discard """
+  targets: "c cpp js"
+  action: compile
+"""
+
+import std/unrolling
```

Comment:
can you also test in VM? if needed, use the fallback not unrolled implementation in VM.

Reply:
It doesn't work in VM, there is a bug in VM

Reply:
```
D:\QQPCmgr\Desktop\Nim\play.nim(7, 18) template/generic instantiation of `unroll` from here
D:\QQPCmgr\Desktop\Nim\lib\system\iterators_1.nim(211, 13) Error: attempting to call routine: 'unroll'
  found system.unroll(a: S, b: T, annotation: static[string]) [iterator declared in D:\QQPCmgr\Desktop\Nim\lib\system\iterators_1.nim(173, 10)]
Traceback (most recent call last)
```

Reply:
use this: (i tested, it works)
```nim
discard """
  targets: "c cpp js"
"""

import std/unrolling
from sugar import collect

template fn() =
  var s: seq[int]
  for i in unroll(0, 100, 5):
    s.add i

  let s2 = collect:
    for i in 0..100: i
  doAssert s == s2

# static: fn() # xxx: Error: cannot generate VM code for for i`gensym0 in pragmaHint(0, 100, "unroll 5"):
fn()
```

Reply:
?

Reply:
I see, thanks